.github/files/setup-wordpress-env.sh

```diff
@@ -129,6 +129,28 @@ for PLUGIN in projects/plugins/*/composer.json; do
 	echo "::endgroup::"
 done
 
+# Install WooCommerce plugin used for some Jetpack integration tests.
+# Todo: check the require paths being used by the Woo tests, they may need updating since the Woo repo structure changed.
+# Todo: what to check so this only runs when Jetpack will be tested?
+# Todo: what PHP_VERSION variable is considered "default"?
+if [[ "$WP_BRANCH" == "latest" && "$PHP_VERSION" == "8.2" ]]; then
```

Comment:
Actually, now that I think of it - if we're using the latest WordPress here from trunk, maybe we can also use WooCommerce from trunk. One caveat is that we'll be introducing another source of possible blockers.

Reply:
No, actually trunk is pretty stable. Judging by the [revision log](https://plugins.trac.wordpress.org/log/woocommerce/trunk), trunk mostly gets updated to RC versions, which may be even better for us - we'll test against WooCommerce code getting ready to be shipped.

Reply:
I think that's still a better solution than building WooCommerce :)

Reply:
@zinigor I've given that a try and updated the PR. So there is now the built Woo plugin directory with the `/tests` dir added manually. Things are not so "plug & play" though 😅 

You can see the recent test failures [here](https://github.com/Automattic/jetpack/actions/runs/5766034900/job/15633214216?pr=32208). Just looking at the first one there:

```
1) WP_Test_Jetpack::test_partner_codes_are_added_to_authorize_url with data set "subsidiary_code" ('subsidiary', 'jetpack_partner_subsidiary_id', 'subsidiaryId')
  Error: Call to undefined method Automattic\Jetpack\Status\Host::get_calypso_env()
  
  /tmp/wordpress-latest/src/wp-content/plugins/jetpack/class.jetpack.php:4557
  /tmp/wordpress-latest/src/wp-includes/class-wp-hook.php:308
  /tmp/wordpress-latest/src/wp-includes/plugin.php:205
  /tmp/wordpress-latest/src/wp-content/plugins/woocommerce/vendor/automattic/jetpack-connection/src/class-manager.php:1858
  /tmp/wordpress-latest/src/wp-content/plugins/jetpack/class.jetpack.php:4514
  /tmp/wordpress-latest/src/wp-content/plugins/jetpack/tests/php/general/test_class.jetpack.php:1041
```

Makes me wonder if there is going to be an issue with the Jetpack packages?